astropy/io/fits/hdu/compressed.py

```diff
@@ -2122,6 +2139,19 @@ def section(self):
         """
         return CompImageSection(self)
 
+    @property
```

Comment:
Could you add a docstring, and same for `compression_type` ?

Reply:
Done